atomic_reactor/plugins/exit_koji_tag_build.py

```diff
@@ -69,6 +72,13 @@ def __init__(self, tasker, workflow, kojihub, target,
         self.target = target
         self.poll_interval = poll_interval
 
+    def _get_koji_session(self):
+        try:
+            return get_koji_session(self.workflow)
+        except KeyError:
+            pass
+        return create_koji_session(self.kojihub, self.koji_auth)
+
```

Comment:
This code is repeated in several other plugins. It should be refactored and moved into koji_util.py